okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -219,7 +219,7 @@ private ConnectionSpec supportedSpec(SSLSocket sslSocket) {
     private String[] tlsVersions;
     private boolean supportsTlsExtensions;
 
-    Builder(boolean tls) {
+    public Builder(boolean tls) {
```

Comment:
Good question. I'd be happy with a zero-arg constructor that defaults to true. It seems that there's no use for the builder in the cleartext case today. Sound good?